async-nats/src/tls.rs

```diff
@@ -71,14 +71,16 @@ pub(crate) struct TlsOptions {
 
 pub(crate) async fn config_tls(options: &TlsOptions) -> io::Result<rustls::ClientConfig> {
     let mut root_store = rustls::RootCertStore::empty();
-    // adds Mozilla root certs
-    root_store.add_server_trust_anchors(webpki_roots::TLS_SERVER_ROOTS.0.iter().map(|ta| {
-        OwnedTrustAnchor::from_subject_spki_name_constraints(
-            ta.subject,
-            ta.spki,
-            ta.name_constraints,
-        )
-    }));
+    for cert in rustls_native_certs::load_native_certs().expect("could not load platform certs") {
```

Comment:
You're panicking here instead of returning Err, map_err instead?

```suggestion
    for cert in rustls_native_certs::load_native_certs().expect("could not load platform certs") {
```

Reply:
Uh, was swapping approach twice and when landing finally on this one forgot to add back map_err. 🥲